policies/blob_reference_property_reserved_segment.rego

```diff
@@ -0,0 +1,48 @@
+package after_resolution
+
+import rego.v1
+
+reserved_segment_name = "blob_ref"
+
+deny contains violation if {
+	group := input.groups[_]
+	group.prefix != null
+	contains_segment(group.prefix, reserved_segment_name)
+	description := sprintf("Registry group '%s' with prefix '%s' contains illegal segment '%s'; reserved for Blob Reference Properties.", [group.id, group.prefix, reserved_segment_name])
```

Comment:
nit: Can you add a URL to link to your blob-reference-properties.md file?

Reply:
Done.